src/JWT/Serializers/JsonSerializerFactory.cs

```diff
@@ -1,16 +1,28 @@
 ﻿namespace JWT.Serializers
 {
-    internal static class JsonSerializerFactory
+    public static class JsonSerializerFactory

```

Comment:
Is it necessary to make it public?

Reply:
It's used by JWT.Extensions.DependencyInjection to get the default serializer